python/sglang/srt/managers/cache_controller.py

```diff
@@ -291,6 +291,14 @@ def __init__(
                 self.storage_backend = MooncakeStore(self.storage_config)
                 self.storage_backend.register_buffer(self.mem_pool_host.kv_buffer)
                 assert self.mem_pool_host.layout == "page_first"
+            elif storage_backend == "aibrix":
+                from sglang.srt.mem_cache.storage.aibrix_kvcache.aibrix_kvcache_storage import (
+                    AibrixKVCacheStorage,
+                )
+
+                self.storage_backend = AibrixKVCacheStorage(
+                    self.storage_config, self.mem_pool_device
```

Comment:
would passing mem_pool_host also work here? asking simply because mem_pool_device is a GPU memory buffer while host memory pool align with this purpose better. Also in a recent refactoring, we introduce a host memory pool registration:
https://github.com/sgl-project/sglang/pull/10300

Reply:
sure , i see mem_pool_device is held by  mem_pool_device , i think it's ok, i will try it